html/static_index.html.haml

```diff
@@ -0,0 +1,76 @@
+!!!
```

Comment:
I think it would be good to do that if possible, otherwise we'll always have to think about doing chances in two places. I'm pretty sure that will be forgotten sooner or later.